src/components/lib/ConnectWalletDialog.js

```diff
@@ -44,8 +66,15 @@ export const ConnectWalletDialog = ({ shown, onConnected }) => {
 	async function chooseWallet(wallet) {
 		setChosenWallet(wallet)
 
+		let connector
+		if (wallet == 'Ledger') {
+			connector = ledgerConnector
+		} else if (wallet == 'Metamask') {
+			connector = injectedConnector
```

Comment:
This may be a good opportunity to use an object to map wallet name -> connector.